nginx/sites-enabled/hashbang-http

```diff
@@ -0,0 +1,18 @@
+# Hashbang server configuration
+# ref: https://gist.github.com/RyanSquared/e59c7e274abef06e63b0e47e46997b00
+#
+server {
+	listen 80;
+	listen [::]:80;
+
+	server_name "~^(?<user>[a-zA-Z0-9]+)\.(?<server>\w{2}\d{1})\.hashbang\.sh$";
+
+	root /home/$user/Public;
+	index index.html;
+
+	location / {
+		if (-f /home/$user/web.sock) {
```

Comment:
Make this `/var/run/http/$user-http.sock`.  We can easily use `pam_exec` to symlink that to `/var/run/${UID}/http.sock`